Makefile.armv7a

```diff
@@ -38,7 +42,9 @@ GCCLIB := $(shell $(CC) $(CFLAGS) -print-libgcc-file-name)
 CRTBEGIN := $(shell $(CC) $(CFLAGS) -print-file-name=crtbegin.o)
 CRTEND := $(shell $(CC) $(CFLAGS) -print-file-name=crtend.o)
 PHOENIXLIB := $(shell $(CC) $(CFLAGS) -print-file-name=libphoenix.a)
-LDLIBS := $(PHOENIXLIB) $(GCCLIB) $(CRTBEGIN) $(CRTEND)
+# The fileter-out enables current toolchains work
```

Comment:
fileter -> filter. This comment is little too cryptic, please explain that LIBSTDCPP is empty if toolchains dooesn't include C++/libstdc++.